examples/chain/8_OfflineSigning/example.go

```diff
@@ -32,7 +32,7 @@ func LoadTxFromFile(fileName string) ([]byte, error) {
 }
 
 func main() {
-	network := common.LoadNetwork("devnet", "")
+	network := common.LoadNetwork("devnet", "lb")
```

Comment:
It is not necessary to provide the node name for `devnet` network. That parameter is only used for testnet and mainnet.

Reply:
actually, it should be testnet. let me update the PR

Reply:
sorry, this is example file. it should not have been altered at all. force-pushing the correct version of the commit.